packages/rum/src/domain/record/serialization/serializeNode.spec.ts

```diff
@@ -588,6 +595,34 @@ describe('serializeNodeWithId', () => {
       })
     })
 
+    it('serializes style node with dynamic CSS that can be fetched with DISABLE_REPLAY_INLINE_CSS false', () => {
+      addExperimentalFeatures([ExperimentalFeature.DISABLE_REPLAY_INLINE_CSS])
+      const linkNode = document.createElement('link')
+      linkNode.setAttribute('rel', 'stylesheet')
+      linkNode.setAttribute('href', 'https://datadoghq.com/some/style.css')
+      isolatedDom.document.head.appendChild(linkNode)
+      Object.defineProperty(isolatedDom.document, 'styleSheets', {
+        value: [
+          {
+            href: 'https://datadoghq.com/some/style.css',
+            cssRules: [{ cssText: 'body { width: 100%; }' }],
+          },
+        ],
+      })
+
+      expect(serializeNodeWithId(linkNode, DEFAULT_OPTIONS)).toEqual({
+        type: NodeType.Element,
+        tagName: 'link',
+        id: jasmine.any(Number) as unknown as number,
+        isSVG: undefined,
+        attributes: {
+          rel: 'stylesheet',
+          href: 'https://datadoghq.com/some/style.css',
+        },
+        childNodes: [],
+      })
+    })
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I would focus on what's expected:

```ts
    it('does not inline external style sheets when DISABLE_REPLAY_INLINE_CSS is enabled', () => {
      addExperimentalFeatures([ExperimentalFeature.DISABLE_REPLAY_INLINE_CSS])
      const linkNode = document.createElement('link')
      linkNode.setAttribute('rel', 'stylesheet')
      linkNode.setAttribute('href', 'https://datadoghq.com/some/style.css')
      isolatedDom.document.head.appendChild(linkNode)
      Object.defineProperty(isolatedDom.document, 'styleSheets', {
        value: [
          {
            href: 'https://datadoghq.com/some/style.css',
            cssRules: [{ cssText: 'body { width: 100%; }' }],
          },
        ],
      })

      expect(serializeNodeWithId(linkNode, DEFAULT_OPTIONS).attributes._cssText).toBeUndefined()
    })

```